### PR TITLE
SQ-64068 [FRONT] - Ajustar cores do NGX-CSS

### PR DESCRIPTION
**Pull request by CaioDalo**
[SQ-64068](https://duopana.myjetbrains.com/youtrack/issue/SQ-64068/FRONT-Ajustar-cores-do-NGX-CSS)

### CODE REVIEW

**Review by JoaoBianco — APPROVED** [review]

**Comment by CaioDalo** [comment]
@coderabbitai review

File: src/helpers/colors.helper.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ export class ColorsHelper {
   getCssVariableValue(variableName: string): string {
     if (this.document?.documentElement) {
       const clearVar = variableName?.replace('var(', '')?.replace(')', '')?.trim()
-      return this.getWindow.window()?.getComputedStyle(this.document?.documentElement).getPropertyValue(clearVar) ?? variableName
+      return this.getWindow.window()?.getComputedStyle(this.document?.documentElement).getPropertyValue(clearVar) || variableName
```
[the remark this answers is not in the document]
Comment (CaioDalo) [reply src/helpers/colors.helper.ts:41]: The operator ?? was it causing color bugs by adding the wrong color to the component once ?? cannot validate empty strings.

**Comment by CaioDalo** [comment]
@coderabbitai review

**Comment by CaioDalo** [comment]
@vinimarcili code review

**Review by vinimarcili — APPROVED** [review]

**Pull request merged**